src/documents/articles/a-quick-polymer-introduction.html.md

```diff
@@ -0,0 +1,160 @@
+
+<blockquote style="margin-top:60px;margin-bottom:30px;font-style:italic;color:grey;font-size:1.5em">I've a feeling we're not in Kansas anymore - Dorothy</blockquote>
+<br/><br/>
+# Web component recap
+
+Web components are a set of standards for creating reusable HTML elements.
+
+For example if you wanted to create an image carousel, you could make a new html element <code>&lt;image-carousel&gt;</code> put all the JS and CSS inside, and use this element anywhere you want.
+
+A key feature is the Shadow DOM, which encapsulates everything inside your element. So in the example above, CSS and JS cannot interfere from the outside preventing JS conflicts and CSS bleed.
+
+# Browser support?
+
+The four areas of web components are:
+* [HTML imports](http://www.w3.org/TR/html-imports/)
+* [HTML templates](http://www.w3.org/TR/html-templates/)
+* [Custom elements](http://www.w3.org/TR/custom-elements/)
+* [Shadow DOM](http://www.w3.org/TR/shadow-dom/).
+
+Browser support figures can be found here — [http://caniuse.com/#search=web%20components](http://caniuse.com/#search=web%20components).
+
+All major browsers have committed support for the v1.0 of the Shadow DOM specification, v1.0 of custom elements and HTML 5 templates.
+
+* [Safari](https://webkit.org/blog/6017/introducing-safari-technology-preview/)
+* [Internet Explorer](https://developer.microsoft.com/en-us/microsoft-edge/platform/status/shadowdom)
+* [Firefox](https://hacks.mozilla.org/2015/11/an-update-on-web-components-and-firefox/)
+
+## HTML imports
+
+The sticking point is HTML imports. Chrome has a native implementation, but [other browsers](http://stackoverflow.com/questions/21177267/es6-modules-vs-html-imports) are looking to Javascript ES6 modules (with a polyfill for HTML imports).
+
+## webcomponents.js
+
+Browser support is varying (at the time of writing) so [webcomponents.js](https://github.com/webcomponents/webcomponentsjs) was made to polyfill the web components API.
+
+This code provides all you need to get started with web components!
+
+# What is Polymer JS?
+
+There are a few wrappers around web components such as [x-tag](http://x-tag.readme.io/) and [bosonic](https://bosonic.github.io).
+
+[Polymer](https://www.polymer-project.org) is a similar wrapper with a few extra goodies thrown in on top.
+
+The aim of the Polymer project is not to create a bloated library, but instead make the smallest library possible, and make the browser do all the work.
+
+\#UseThePlatform is the name Polymer give to represent the want for the browser to do all the heavy lifting without libraries.
+
+One of the advantages of using Polymer is that it has a built in data binding model (like [React](https://facebook.github.io/react/) or [Angular](http://angularjs.org/)) enabling complete applications to be made without any other libraries.
+
+Polymer [supports evergreen browsers](https://www.polymer-project.org/1.0/docs/browsers) and when combined with webcomponents.js supports IE 11+.
```

Comment:
Do we support all 'evergreen browsers' or all major ones? Also we need to reword this because according to the `webcomponentsjs` README we still have partial support for IE10. Has this changed? cc @addyosmani @zenorocha